### DISCUSSION TITLE
AppRouter is not working with optional catch-all segments in PageRouter

### DISCUSSION THREAD

**Discussion by voronianski** [vercel/next.js#62677] [App Router]
hey all, please check this example and steps in demo repo -
https://github.com/voronianski/nextjs-page-router-with-app-router-issues?tab=readme-ov-file#approuter-is-not-working-with-optional-catch-all-segments-in-pagerouter

Should this be considered a bug or am i missing something?

Thanks

**Comment by tangye1234** [comment]
The latest canary still exists, 14.2.0-canary.6
